README.md

```diff
@@ -60,16 +60,15 @@ This package allows you to create data transfer object definitions, classes, whi
 We did our best to keep the syntax and overhead as little as possible:
 
 ```php
+use \App\Models\Author;
```

Comment:
Imports are usually written without leading `\` as they're always qualified from the "root namespace 😄 Applies to the other examples too:
```suggestion
use App\Models\Author;
```

Reply:
Yeah, I am aware. Since that was how it was written in the original, I did not want to be too opinionated.

Reply:
Took a look at other Spatie packages and found that this scenario of prefixing a use statement with a backslash is unique. I'll make another commit to remove this.

Reply:
Resolved by eb0f5ad45c211bff4488468673d4147035bfb802